esports_data_from_rimble/Makefile

```diff
@@ -1,4 +1,4 @@
-default: run
+default: match-winner
```

Comment:
`run` is not actually a target in this makefile (though it is in the other examples). I update to default to `match-winner` target